gulpconfig.json

```diff
@@ -16,5 +16,20 @@
 		"dev": "./src",
 		"dist": "./dist",
 		"distprod": "./dist-product"
-	}
+	},
+	"colors": [
```

Comment:
What about primary, secondary, light, dark? I'm not sure about the alert colors. The CSS is compiled for all the colors in the colors and the theme colors map.